src/main/java/com/techsophy/tsf/runtime/form/changelog/ElasticFieldMigration.java

```diff
@@ -7,24 +7,32 @@
 import io.mongock.api.annotations.RollbackExecution;
 import lombok.RequiredArgsConstructor;
 import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Value;
 import org.springframework.data.mongodb.core.MongoTemplate;
 import org.springframework.data.mongodb.core.query.Query;
 import org.springframework.data.mongodb.core.query.Update;
+
+import static com.techsophy.tsf.runtime.form.constants.FormModelerConstants.ELASTIC_ENABLE;
 import static com.techsophy.tsf.runtime.form.constants.FormModelerConstants.EXECUTION_IS_FAILED;
 
 @ChangeUnit(id="push-to-elastic",order = "5")
 @Slf4j
 @RequiredArgsConstructor
 public class ElasticFieldMigration
 {
+    @Value(ELASTIC_ENABLE)
```

Comment:
pls dont add the annotations via constants

Reply:
Yes replacing with string only